### ISSUE TITLE
Blank homepage on logged out accounts

### ISSUE THREAD

**Issue by HARANARATE** [Rehike/Rehike#557]
### Rehike version

0.8.0

### Operating system

Windows 10

### PHP version

PHP 8.2.12

### What's going on?

![image](https://github.com/Rehike/Rehike/assets/148234876/258de3ef-6a7d-4722-80ac-fd36557e53e0)

**Comment by Xammand** [comment]
I think the homepage is blank because if you go on youtube (not using Rehike) signed out, it will just show nothing but a thing saying to watch videos to get suggestions while signed out.

**Comment by kawapure** [comment]
Parts of this issue were duplicates of #227, so they have been removed.

The homepage issue is improved in ae2b4f2. Now there will be a renderer, but I'm going to keep this issue open until this issue is completely resolved. It should be populated, if possible.

**Comment by kawapure** [comment]
95f0107 theoretically fixes this homepage issue, but it takes ~15 seconds to go through, which is too long for one request to wait on. I think it's solved to the most of its abilities, apart from a loading screen being implemented along with automatic refresh.
